### DISCUSSION TITLE
Installation of Sunbird-Lern-Group Service

### DISCUSSION THREAD

**Discussion by devaj04** [Sunbird-Lern/Community#94] [Installation] [answered]
**thisActorSystem** - what we have put (maybe system name)
 **play secret key**  - from where we can get the secret key

<img width="953" alt="image" src="https://user-images.githubusercontent.com/125238862/227155358-6336460b-322b-40c8-a195-99b2602f2972.png">

After hitting the health check API - **http://localhost:9000/health**
**Getting a response is empty**
{
    "id": "",
    "ver": "v1",
    "ts": "2023-03-23 14:32:06:882+0530",
    "params": null,
    "result": {
        "Response": {},
        "healthy": true
    },
    "responseCode": 200
}

**Comment by reshmi-nair** [comment]
@devaj04 I will check on this and update soon

**Reply by reshmi-nair** [reply]
@devaj04 /service/application.conf has a thisActorSystem {} object which can hold all the actors with in the service. So pls check the existing actors like 'CreateGroupActor', 'ReadGroupActor' configurations in application.conf file. If you have any new actors add to this list.

"play.http.secret.key"  is also already mentioned in application.conf. When you create a new microservice with play framework you can give a random string as your secret key.
Have you setup the Cassandra keyspace and redis?

**Reply by devaj04** [reply]
@reshmi-nair  Yes, I have set up Cassandra Keyspace.
Still getting a null response.

**Reply by reshmi-nair** [reply, marked as answer]
@devaj04 Its not a null response, it is an empty response with healthy flag as true. That means you have successfully setup groups. Now you can try create group API and other APIs.
"result": {
"Response": {},
"healthy": true
},

**Comment by devaj04** [comment]
@reshmi-nair 
Thank you
I will try to create Group API if the issue came will create a different thread.
